sdk/nodejs/runtime/resource.ts

```diff
@@ -282,19 +282,18 @@ async function resolveOutputs(res: Resource, t: string, name: string,
 /**
  * registerResourceOutputs completes the resource registration, attaching an optional set of computed outputs.
  */
-export function registerResourceOutputs(res: Resource, outputs: Inputs) {
+export function registerResourceOutputs(res: Resource, outputs: Input<Inputs>) {
```

Comment:
you should also update hte signature of ComponentResource.registerOutputs.  Otherwise TS will complain if people actually try to do this.

Reply:
Ah, good catch, will do.